etcdctl/README.md

```diff
@@ -801,6 +801,254 @@ The provided transformer should read until EOF and flush the stdout before exiti
 finished transforming keys
 ```
 
+### AUTH <enable or disable>
+
+`auth enable` activates the authentication. `auth disable` to deactivate. Once authentication is on, every client request will be checked on its permission based on its user name.
```

Comment:
`auth enable` activates authentication on an etcd cluster and `auth disable` deactivates. When authentication is enabled, etcd checks all requests for appropriate authorization.